lib/modem_info/modem_info_json.c

```diff
@@ -171,6 +171,10 @@ static int device_data_add(struct device_param *device, cJSON *json_obj)
 	total_len += json_add_str(json_obj, "appVersion", device->app_version);
 	total_len += json_add_str(json_obj, "appName", device->app_name);
 
+	if (device->bat_charge < 0xFF) {
```

Comment:
I'm slightly confused here - states that this value indicates the percentage battery charge level. Why `255` then? I hope we do not expect the battery to be charged for more than 100% :D